app/src/main/java/moe/apex/rule34/preferences/Pref.kt

```diff
@@ -452,6 +452,10 @@ enum class ImageSource(override val description: String, val site: ImageBoard) :
     R34("Rule34", Rule34);
 
     companion object {
+        fun get(name: String): ImageSource? {
```

Comment:
Probably not needed if we can directly pass in the ImageSources to the destinations.